frontend/appflowy_flutter/lib/workspace/presentation/home/menu/view/view_item.dart

```diff
@@ -215,8 +218,11 @@ class _SingleInnerViewItemState extends State<SingleInnerViewItem> {
       ),
       buildWhenOnHover: () => !widget.showActions,
       builder: (_, onHover) => _buildViewItem(onHover),
-      isSelected: () => (widget.showActions ||
-          getIt<MenuSharedState>().latestOpenView?.id == widget.view.id),
+      isSelected: () =>
+          (widget.isDraggable ||
+              widget.categoryType == FolderCategoryType.favorite) &&
+          (widget.showActions ||
+              getIt<MenuSharedState>().latestOpenView?.id == widget.view.id),
```

Comment:
@LucasXu0, this logic change in last commit didn't incorporate favorites, since draggable is not yet available for favorites it would always be false thus breaking selection and menu, I have added the condition